src/ota.c

```diff
@@ -12,7 +12,7 @@
 #include "coap_client.h"
 #include <golioth/golioth_debug.h>
 #include "golioth_util.h"
-#include "zcbor_utils.h"
+#include <golioth/zcbor_utils.h>
```

Comment:
What is the reason to expose that header publicly? (There is no reasoning in the commit message.)

Reply:
So that applications can use it. In this case, I'm using it in the lightdb/get example for CBOR.